media/server/gstplayer/CMakeLists.txt

```diff
@@ -20,6 +20,10 @@
 # Find includes in corresponding build directories
 set( CMAKE_INCLUDE_CURRENT_DIR ON )
 
+add_compile_options(
```

Comment:
maybe add a comment with explanation and reference to jira ticket which should remove this part of code?